worlds/sc2/Options.py

```diff
@@ -231,6 +258,131 @@ class MaxNumberOfUpgrades(Range):
     default = -1
 
 
+
+class IncludeMutations(Range):
+    """Determines how many of the 3 mutations for the 7 units that have them can appear."""
+    display_name = "Include Mutations"
+    range_start = 0
+    range_end = 3
+    default = 1
+
+
+class IncludeStrains(Range):
+    """Determines how many of the 2 strains for the 7 units that have them can appear."""
+    display_name = "Include Strains"
+    range_start = 0
+    range_end = 2
+    default = 1
+
+
+class Kerriganless(Choice):
+    """Determines whether Kerrigan is playable outside of missions that require her.
+
+    Off:  Kerrigan is playable as normal.
+    On:  Kerrigan is not playable.  Other hero units stay playable,
+    and locations normally requiring Kerrigan can be checked by any unit.
+    Kerrigan level items, active abilities and passive abilities affecting her will not appear.
+    On Without Passives:  In addition to the above, Kerrigan's passive abilities affecting other units (such as Twin Drones) will not appear."""
+    display_name = "Kerriganless"
+    option_off = 0
+    option_on = 1
+    option_on_without_passives = 2
+
+
+class KerriganChecksPerLevelPack(Range):
+    """Determines how many locations need to be checked for a level pack to be received.  Missions have between 4 and 5 locations each.  
+    Kerrigan's maximum level is always 70."""
+    display_name = "Checks Per Kerrigan Level Pack"
+    range_start = 1
+    range_end = 10
+    default = 1
+
+
+class KerriganCheckLevelPackSize(Range):
+    """Determines how many levels Kerrigan gains when enough locations are checked.  Kerrigan's maximum level is always 70."""
+    display_name = "Check Level Pack Size"
+    range_start = 0
+    range_end = 5
+    default = 0
+
+
+class KerriganLevelItemSum(Range):
+    """Determines the sum of the level items in the seed.  This does not affect levels gained from checks.  
+    Kerrigan's maximum level is always 70."""
+    display_name = "Kerrigan Level Item Sum"
+    range_start = 0
+    range_end = 140
+    default = 70
```

Comment:
The real max level is 25000 in the new mod (insanely high)

Reply:
I removed the mention of the max level, was that the desired change?

Reply:
it's not hardcoded to be 70. We can make a level cap in the future